### PR TITLE
Fixes replayed bundle being reversed

### LINKED ISSUE #74: replayBundle sends trytes in reverse order
When invoking the `replayBundle` command, the resulting trytes are broadcast in reverse order, preventing `getBundles` from working correctly.

Example:  
https://thetangle.org/bundle/GMMGJTEWAINANUTNRNQLUEZFTTTSEPODSODQFJWCPGBMFZHXL9ZLJXBDMDZKCKERCOZRRCXCP9AQLIG9X

Bundle should look like this (note reverse order):  
https://thetangle.org/bundle/EYTSBAEABMXUNDVOMXBGPZWVGXTXGYHENBAMEETOSHXMNXUBSTUOUXOLVOFRCBXBZXIOMVWRPSYYEFEVX

### PR DESCRIPTION
**Pull request by normpad**
Fixes #74

### CODE REVIEW

**Comment by todofixthis** [comment]
Hey @normpad thanks for submitting this pull request!

Could you include a unit test for regressions (i.e., if somebody later goes back and reverses the order of transactions again, the unit test should fail)?

The fact that it took us so long to catch this bug manually makes me concerned; I would sleep a lot better at night if we had an automated test for it (:

**Comment by normpad** [comment]
Absolutely, I'm just not entirely sure how to go about that. But assuming trytes should always be passed to attachToTangle tail to head, wouldn't it be fitting to have a check before passing the trytes to the node and throw an error if they are out of order?

**Comment by todofixthis** [comment]
Hm.  Interesting.

Here's what I think you can do:

Modify this line:  `test/commands/extended/replay_bundle_test.py:599`
```python
mock_send_trytes = mock.Mock(return_value=send_trytes_response)
```

Change it to something like this:
```python
def mock_send_trytes(request):
  """
  Ensures that the correct trytes are sent to the ``sendTrytes`` command
  to replay the bundle.

  References:
    - https://github.com/iotaledger/iota.lib.py/issues/74
  """
  self.assertEqual(
    request['trytes'],

    [
      ... transaction trytes go here ...
    ]
  )

  return send_trytes_response

...

  with mock.patch(
      'iota.commands.extended.send_trytes.SendTrytesCommand._execute',
      mock_send_trytes,
  ):
    ...
```

**Comment by todofixthis** [comment]
Incidentally, once https://github.com/iotaledger/iota.lib.py/issues/43 is implemented, we can set up a real private subtangle when running unit tests, and then we can get rid of all of this mocking nonsense 😸

File: test/commands/extended/replay_bundle_test.py [hunk 1]
```diff
@@ -593,10 +593,20 @@ def test_happy_path(self):
...
+    def mock_send_trytes(self,request):
+      """
+      Ensures that the correct trytes are sent to the ``sendTrytes`` command
+      to replay the bundle.
+
+      References:
+        - https://github.com/iotaledger/iota.lib.py/issues/74
+      """
+      from unittest import TestCase
+      TestCase().assertEqual(request['trytes'], send_trytes_response['trytes'])
```
Comment (todofixthis) [inline test/commands/extended/replay_bundle_test.py]: Python 2.7 doesn't like that so much.  Maybe something like this:

```python
def mock_send_trytes(_, request):
  ...
  self.assertEqual(...)
```

Comment (todofixthis) [reply test/commands/extended/replay_bundle_test.py]: Aye.  I admit; it feels like a hack, either way, but that's why I'm so keen on getting #43 set up (:

**Comment by todofixthis** [comment]
Looks great!  Merging now.

As far as I'm aware, this was the last obstacle to releasing v2.0.0 stable (:  Thanks!!!

**Pull request merged**